schema/ssobject.avsc

```diff
@@ -0,0 +1,87 @@
+{
+	"namespace": "lsst",
+	"name": "ssObject",
+	"type": "record",
+	"fields": [
+		{"name": "ssObjectId", "type": "long"},
+		{"name": "oe", "type": [{"type": "record",
+			"name": "oe", "fields": [
+				{"name": "q", "type": ["double", "null"]},
+				{"name": "e", "type": ["double", "null"]},
+				{"name": "i", "type": ["double", "null"]},
+				{"name": "lan", "type": ["double", "null"]},
+				{"name": "aop", "type": ["double", "null"]},
+				{"name": "M", "type": ["double", "null"]},
+				{"name": "epoch", "type": ["double", "null"]}
+				]}, "null"]},
+		{"name": "oeCov", "type": [{"type": "record",
+			"name": "oeCov", "fields": [
```

Comment:
Why are there no diagonal elements in this covariance matrix (I know the DPDD says there are only 21 elements; is that an error in the DPDD)?